src/dqrobotics/interfaces/coppeliasim/DQ_CoppeliaSimInterfaceZMQ.cpp

```diff
@@ -765,7 +756,7 @@ void DQ_CoppeliaSimInterfaceZMQ::set_joint_torques(const std::vector<std::string
 double DQ_CoppeliaSimInterfaceZMQ::_get_joint_torque(const int &handle) const
 {
     _check_client();
-    return _ZMQWrapper::get_sim()->getJointForce(handle);
+    return -_ZMQWrapper::get_sim()->getJointForce(handle);
```

Comment:
@juanjqo I think GitHub interface isn't showing everything, but if there's a `-` sign here it's probably best to have some additional explanation.

Reply:
Hi @mmmarinho.  I agree. I'll add the comments. I think there is a bug in the `getJointForce` ZMQ remote API method. I open an [issue](https://forum.coppeliarobotics.com/viewtopic.php?p=41690#p41690) in the CoppeliaSim Forum.